.github/workflows/ci_skipped.yaml

```diff
@@ -1,34 +0,0 @@
-name: CI Skipped
```

Comment:
Im going to remove required checks since they are part of the merge queue anyway, so a PR won't be merged without them